templates/CommentsInterface.ss

```diff
@@ -1,4 +1,4 @@
-<% require themedCSS('comments', 'comments') %>
+<% require themedCSS('client/dist/styles/comments', 'comments') %>
```

Comment:
The second parameter is meant to be a media  attribute. I'm pretty sure `comments` is a nonsensical value here. However, if I remove it, there's a good chance we'll end up injecting CSS in people front end without them noticing.